### PR TITLE
Added description how to run the app locally

### PR DESCRIPTION
**Pull request by advename**
Moreover, I implemented the app logo, it's always more nicer to have something visual in a readme :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,39 @@
 # TypeOfNaN JavaScript Quizzes
```
Comment (nas5w) [inline README.md]: Let's remove this line since we'll have the logo

File: README.md [hunk 2]
```diff
@@ -1,9 +1,39 @@
...
 
+![TypeOfNaN Logo javascript quizzes](typeOfNaN-logo.jpg)
```
Comment (nas5w) [inline README.md]: Can we change to:

`TypeOfNaN JavaScript Quizzes`

File: README.md [hunk 3]
```diff
@@ -1,9 +1,39 @@
...
+
+### 1. Clone the repo
+
+Run this command to clone the repo
+
+`git clone https://github.com/nas5w/typeofnan-javascript-quizzes`
+
+### 2. Install dependencies
+
+The first before you can use the app, you have run this command to install all the dependencies
```
Comment (nas5w) [inline README.md]: Remove "The" from beginning of line. This line should be:

First, before you can use the app, you have run this command to install all the dependencies

File: README.md (hunk 2 above)
Comment (nas5w) [reply README.md]: Also, is it possible to make the logo a bit smaller? Maybe between 50-75% width of the readme file?

Comment (advename) [reply README.md]: Sure, i have the file already ready, but need to create another pull request to update the file

**Pull request merged**